src/State/ProcessorInterface.php

```diff
@@ -26,5 +26,5 @@ interface ProcessorInterface
     /**
      * Handle the state.
      */
-    public function process($data, Operation $operation, array $uriVariables = [], array $context = []);
+    public function process(mixed $data, Operation $operation, array $uriVariables = [], array $context = []);
```

Comment:
Is there any use case for returning something? Or should there be a void return type?

Reply:
Yes there is: you return your resource, for instance after a `POST` to obtain the generated identifier.

Reply:
Makes sense, thanks